app/assets/stylesheets/admin/components/page_actions.css.scss

```diff
@@ -0,0 +1,7 @@
+.page-actions {
+  li {
```

Comment:
isn't there a class attribute we could use instead? As far as I know, coupling to a particular DOM element is not very changeable.

Reply:
no, the li entries don't have any class in the spree code. I think this is fine. page actions are a ul with li entries, and we want to change all those li entries.

Reply:
This is how it would look in narrower screens, or using languages where the translation is even longer:

![20190328031740-header-buttons-medium-screens](https://user-images.githubusercontent.com/2243/55106136-20e5d700-5109-11e9-848a-1143fa904c51.png)

Changing the `display` property of the button from `inline` to `inline-block` could really help make this more responsive. I recommend this instead:

```css
.page-actions {
  li {
    margin-top: 1px;
    margin-bottom: 1px;
  }

  a {
    display: inline-block;
  }
}
```

The 1px margin (because we're not trying to be pixel-perfect) is to make sure there is space between rows of buttons.

Then this is how the buttons would look:

![20190328033019-header-action-buttons-inline-block](https://user-images.githubusercontent.com/2243/55106473-019b7980-510a-11e9-9733-dc8e37a86ea6.png)

Reply:
yeah, thanks @kristinalim 
now I ask myself, why did I use padding and not margin?
the inline-block tip is 💯 
I replaced my commit with your code 👍